move/gas_service/sources/versioned/gas_service_v0.move

```diff
@@ -123,18 +133,18 @@ module gas_service::gas_service_v0 {
     }
 
     #[test_only]
-    public(package) fun balance(self: &GasService_v0): &Balance<SUI> {
-        &self.balance
+    public(package) fun balance<T>(self: &GasService_v0): &Balance<T> {
```

Comment:
Let's make this getter public since it'll be useful now with the switch to a Bag especially?